### PR TITLE
Upgrade GDAL from v3.6.2 to v3.6.3 (EL9)

### PR DESCRIPTION
**Pull request by hummeltech**
https://github.com/OSGeo/gdal/releases

### CODE REVIEW

**Comment by hummeltech** [comment]
Waiting for #123, #125 & #127 to be tested and merged before merging.

**Review by jbronn — APPROVED** [review]
LGTM

**Pull request merged**
